### PR TITLE
Molecule v2

### PR DESCRIPTION
**Pull request by sbesson**
This PR makes the necessary changes to test this role with Ansible 2.4 and above and Molecule 2.

- e25f8bc  creates the Molecule 2 initial setup using `molecule init scenario -r $(basename $PWD) -s default -d docker`
- f74e932  fixes the yamllint warnings and errors reported by `molecule test` and u
-  5f26f55  upgrades tests to be compliant with Molecule 2/testinfra 1.14

With openmicroscopy/ome-ansible-molecule-dependencies#10, this PR should be tested by Travis against Ansible 2.4 in the daily https://travis-ci.org/snoopycrimecop/ansible-roles build.

Proposed tag: `2.1.2`

### CODE REVIEW

File: molecule/default/INSTALL.rst [hunk 1]
```diff
@@ -0,0 +1,26 @@
+*******
```
Comment (manics) [inline molecule/default/INSTALL.rst]: Can you delete this file?

**Review by manics — CHANGES_REQUESTED** [review]

File: molecule/default/INSTALL.rst (hunk 1 above)
Comment (sbesson) [reply molecule/default/INSTALL.rst]: :+1: b649bb5

File: molecule/default/tests/test_default.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import os
+
+import testinfra.utils.ansible_runner
+
+testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
+    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('all')
+
+
+def test_hosts_file(host):
```
Comment (manics) [inline molecule/default/tests/test_default.py]: Missed this in my first review, looks like this default test has crept in.

Comment (sbesson) [reply molecule/default/tests/test_default.py]: See 69995fb

**Review by manics — APPROVED** [review]

**Comment by sbesson** [comment]
Now deployed to https://galaxy.ansible.com/openmicroscopy/ice

**Pull request merged**